paraatm/simulation_method/vcas.py

```diff
@@ -145,19 +148,19 @@ def simulation(self, input_cmd=None):
             ignore_index=True)  # end cmd
 
         # self.init_NATS()
-        self.environmentInterface.load_rap(os.environ.get('NATS_HOME') + '/' + 'share/tg/rap')  # default wind file
+        self.environmentInterface.load_rap(os.environ.get('GNATS_HOME') + '/' + 'share/tg/rap')  # default wind file
```

Comment:
For me, this does not work because I have `GNATS_HOME` pointing to the `GNATS_Standalone` directory, not `GNATS_Server`.  Based on the installation instructions (in particular, the GNATS examples instructions says to put the examples in the `GNATS_Standalone` directory), this is my understanding of how `GNATS_HOME` should be set up to work with Python.

Also, `GnatsEnvironment` stores a reference to the `share` directory for you, so you can use this:
```suggestion
        self.environmentInterface.load_rap(os.path.join(GnatsEnvironment.share_dir, 'tg', 'rap'))  # default wind file
```
One advantage of this is that it will work when the user wants to override `GNATS_HOME` by using the `gnats_home` argument to `start_jvm`.